src/OpenApi/sample/Transformers/AddBearerSecuritySchemeTransformer.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.OpenApi;
+using Microsoft.OpenApi.Models;
+
+public sealed class BearerSecuritySchemeTransformer(IAuthenticationSchemeProvider authenticationSchemeProvider) : IOpenApiDocumentTransformer
+{
+    public async Task TransformAsync(OpenApiDocument document, OpenApiDocumentTransformerContext context, CancellationToken cancellationToken)
+    {
+        var authenticationSchemes = (authenticationSchemeProvider is not null)
+                ? await authenticationSchemeProvider.GetAllSchemesAsync()
+                : [];
+        var requirements = authenticationSchemes
+                .Where(authScheme => authScheme.Name == "Bearer")
+                .ToDictionary(
+                    (authScheme) => authScheme.Name,
```

Comment:
Didn't we just force this to be `"Bearer"`?

Reply:
Yep -- I'll tweak this a bit since the [copy-pasta risk is high](https://github.com/dotnet/aspnetcore/pull/54935#discussion_r1552299418).

Reply:
But why `ToDictionary` if there's only one key?